docs/tutorials/mongoid-documents.txt

```diff
@@ -176,6 +177,38 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
+The StringifiedSymbol Type
+--------------------------
+
+When using the built-in Symbol field type, Mongoid defaults to serializing the field into the BSON::Symbol
+type. For more information on the BSON::Symbol type, see
+`here <https://docs.mongodb.com/mongoid/master/tutorials/mongoid-documents/#bson-symbol-type>`_.
+BSON::Symbol is deprecated, so the StringifiedSymbol type allows users to write code
+that makes use of the built-in Symbol type while ensuring interoperability with other drivers.
+The StringifiedSymbol type stores all data on the database as a String, while returning values to the
+user as a Symbol.
+
+An example usage is shown below:
+
+.. code-block:: ruby
+
+  class Post
+    include Mongoid::Document
+
+    field :status, type: StringifiedSymbol
+  end
+
+  post = Post.new(status: "hello")
+  post.status # => :hello
+
+All values will be stringified upon being sent to the database, and all values will be converted
+to symbols when returned to the user. This includes values that cannot be converted directly to
```

Comment:
user --> application

Reply:
And could you be more specific about what would happen to values that cannot be converted to symbols?

Reply:
I think adding an example where `42` is assigned to the status and the returned value would be `:42` would do the trick.

Reply:
The way we have mongoize and demongoize implemented right now (`object.to_s` and `object.to_s.to_sym`, respectively) results in `:"42"` being returned, so that's what I've shown in the new example.